### PR TITLE
Remove the generic type from the extends tag

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -136,9 +137,14 @@ exports.astNodeVisitor = {
...
             }
             if (node.superClass) {
-              // Add class inheritance information because JSDoc does not honor
-              // the ES6 class's `extends` keyword
-              if (leadingComment.value.indexOf('@extends') === -1) {
+              const extendsIndex = lines.findIndex(line => line.includes('@extends'));
+              if (extendsIndex !== -1) {
+                // Remove the generic type from the extends tag, for example:
+                // "@extends {ParentClass<GenericType>}" -> "@extends {ParentClass}"
+                lines[extendsIndex] = lines[extendsIndex].replace(extendsWithGenericTypeRegEx, '@extends {$1}');
```
Comment (ahocevar) [inline index.js]: Couldn't you just remove the line with `@extends` here, and always add the `@extends` derived from the class's `extends` keyword below?

Comment (fredj) [reply index.js]: yes, good suggestion

Comment (fredj) [reply index.js]: done

**Review by ahocevar — APPROVED** [review]
Thanks! Bonus if you also bump the version and publish to npm.

**Pull request merged**